app/src/containers/CheckoutPage.tsx

```diff
@@ -158,8 +164,11 @@ class CheckoutPage extends React.Component<RouteComponentProps, CheckoutPageStat
   }
 
   fetchOrderData() {
+    const { match } = this.props;
+    const cartCode = match.params.cart;
+    const cart = cartCode || 'default';
     login().then(() => {
-      cortexFetch(`/?zoom=${zoomArray.sort().join()}`,
+      cortexFetch(`/carts/${Config.cortexApi.scope}/${cart}?zoom=${zoomArray.sort().join()}`,
```

Comment:
@rostyk-kanafotskyy Unfortunately this will not work. `carts` is only available on EP 7.6 and up. It also may not even be available if multi-cart is disabled on 7.6, but `defaultcart` will be.

Reply:
Another approach could be to request `defaultcart`  and `carts` through the zoom, and check if `carts` is actually returned, then act on that.

Reply:
Discussed over slack. Thinking about it more, we can merge as the resource names should be the same. We'll keep an eye on this.